implementations/rust/daemon/src/bin/ockamd.rs

```diff
@@ -0,0 +1,26 @@
+use std::io::stdin;
+
+use ockamd::args::Args;
+
+fn main() {
+    let args = Args::parse();
+    println!("args: {:?}", args);
+
+    // read stdin for messages to encrypt and route
+    let input = stdin();
+    let mut buf = String::new();
+
+    loop {
+        match input.read_line(&mut buf) {
```

Comment:
I have a better opinion on how to do this. But that can come in future PR.

Reply:
For example, we should allow input to be passed via a pipe, atty, file, or channel.